testdata/baselines/reference/submodule/conformance/typeFromContextualThisType.symbols.diff

```diff
@@ -6,7 +6,7 @@
  >this.b : Symbol(b, Decl(bug25926.js, 0, 23))
 ->this : Symbol(__type, Decl(bug25926.js, 0, 11))
 ->b : Symbol(b, Decl(bug25926.js, 2, 9))
-+>this : Symbol(o1, Decl(bug25926.js, 0, 11))
++>this : Symbol(�type, Decl(bug25926.js, 0, 11))
```

Comment:
This is getting some sort of goofy symbol name, it seems.

Reply:
this is the usual string for the intenral name of types. Does it not normally print?